### ISSUE TITLE
[Question] How to implement loading indicator in the Home screen

### ISSUE THREAD

**Issue by suho** [tquocit/survey-demo#2]
## Issue

Currently, there are no loading indicators on the Home screen, which is different from the design.

![image](https://github.com/tquocit/survey-demo/assets/19943832/52cd9ae1-e9b7-4fc3-8ea4-a73c721aeda1)

Could you tell me how to implement the loading indicators like the design?

**Comment by tquocit** [comment]
Yes. For this one, I planned to improve after complete the main features. To implement this one
1. We can create UIView+Extensions, and feature showShimmerAnimation. To implement this one, we can create a gradientLayer with CAGradientLayer with corner radius and 2 color. Then we use CABasicAnimation with key "location" and let animation run repeat .infinity 
2. Second solution is we can use a library that I used before: [UIView-Shimmer](https://github.com/omerfarukozturk/UIView-Shimmer)
